frontend/README.md

```diff
@@ -0,0 +1,48 @@
+Install following dependencies:
+$ pip3 install getpass gradio pynrok cv2
+
+To launch simple gradio app locally, open a terminal and run
+$ cd path/to/fsdl-2022-3D-reconstruction/frontend/
+$ python3 make_gradio.py
```

Comment:
When you use the proper conda env the correct command will be python and not python3. Can you update when you are using the proper env?